src/Actions/Trackers/FocusTracker.cs

```diff
@@ -1,5 +1,7 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information
```

Comment:
It looks like this is triggering the license check to fail... I think it might be the missing period at the end?